LICENSE

```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2010-2017 Chris Kirkham <chrisjameskirkham@gmail.com>
+	      		Richard Barlow <richard@richardbarlow.co.uk>
```

Comment:
It'd be good if we could have consistent whitespace here (I suggest using spaces since we want to align with the top line).